src/VisualStudio/Core/Def/Implementation/DesignerAttribute/VisualStudioDesignerAttributeService.cs

```diff
@@ -108,7 +108,7 @@ private async Task StartAsync(CancellationToken cancellationToken)
         private async Task StartWorkerAsync(CancellationToken cancellationToken)
         {
             _workQueue = new AsyncBatchingWorkQueue<DesignerAttributeData>(
-                TimeSpan.FromSeconds(1),
+                TimeSpan.FromMilliseconds(500),
```

Comment:
what is the rationale for this change?

Reply:
i actually reduce it even more in the next PR.  the point of these delays is to allow us to hear about a large set of result from oop at once, and then report them at once to VS in one go.  i.e. we don't want to hear about 100 files, and then send 100 calls to update VS (each which might hit the UI thread).  Instead, it's better to hear about 100 files, then report it in one go to VS on the UI thread once.

So the goal is simply to give enough time to hear about a lot, but not so long that it feels slow to the user for thigns to update.  250ms seems to work well.  It allows for hearing about tons of files, not saturating the UI thread, and providing low latency between hearing about changes and updating the UI with them.  Note: my next PR adds a comment on this.

Reply:
"250ms seems to work well."

The PR is changing it to 500ms though?

Reply:
next pr reduces it further :)